xcom2-launcher/xcom2-launcher/Forms/SettingsDialog.cs

```diff
@@ -12,6 +12,7 @@ namespace XCOM2Launcher.Forms
     public partial class SettingsDialog : Form
     {
         protected Settings Settings { get; set; }
+        private readonly List<string> ModPaths;
```

Comment:
Why is this needed? It seems to just follow `modPathsListbox.Items`

Reply:
True. But ListBox.ObjectCollection is a pain to work with. I prefer to avoid Items.Cast<string>().ToList() stuff all the time. Using a second list just seemed to be the lesser evil to me in this case.

Reply:
Ah I see your point. Could you make separate helper functions for adding/removing entries then that will be responsible for keeping the 2 in sync?

Also, please document your reasoning in the code (probably a comment next to the field declaration)

Reply:
I decided to removed the extra list after all. It wasn't worth keeping it to save two casts.